db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/SchedulerBuilder.java

```diff
@@ -91,8 +92,18 @@ public <T extends Task<?> & OnStartup> SchedulerBuilder startTasks(List<T> start
         return this;
     }
 
+    public SchedulerBuilder taskResolver(TaskResolver taskResolver) {
+        this.taskResolver = taskResolver;
+        return this;
+    }
```

Comment:
I am uncertain what the use-case for this is. If you do not use the default or a variant of it not much will work

Reply:
I see you have commented a bit on it. We can take the discussion in another PR